salt/environment_settings.yml

```diff
@@ -123,12 +124,14 @@ environments:
       rabbitmq:
         role_prefixes:
           - edxapp
-          - ecommerce
```

Comment:
Why was this removed?

Reply:
Because we aren't going to be using the ecommerce app any more